pkg/util/utils.go

```diff
@@ -15,7 +15,13 @@ import (
 	"github.com/youmark/pkcs8"
 )
 
-const LegacyPem = "legacy-pem"
+const (
+	LegacyPem        = "legacy-pem"
+	HeaderTpplApikey = "tppl-api-key"
+	OauthTokenType   = "Bearer"
+	HeaderOauthToken = OauthTokenType + " "
```

Comment:
I don't think you need this constant at all. You can make the string placeholder `"%s %s"` and use OauthTokenType and the token value where needed

Reply:
got it. Removed `HeaderOauthToken` in favor of directly using `OauthTokenType` instead. Done by latest commit